mne_bids/tests/test_mne_bids.py

```diff
@@ -151,11 +164,31 @@ def test_vhdr():
 
     raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
                 task=task, raw_file=raw_fname, output_path=output_path,
-                overwrite=True, kind='eeg')
+                overwrite='overwrite', kind='eeg')
 
     cmd = ['bids-validator', '--bep006', output_path]
     run_subprocess(cmd, shell=shell)
 
+    # check the 'clear' overwrite command works correctly
+    raw_to_bids(subject_id=subject_id, session_id=session_id, run=run2,
+                task=task, raw_file=raw_fname, output_path=output_path,
+                overwrite='clear', kind='eeg')
+    # we will need to go over the tree to ensure there are no files with
+    # `run-01` in their file names:
+    for _, _, files in os.walk(output_path):
+        for f in files:
+            if 'run-%s' % run in f:
+                raise OSError(EEXIST)
+
+    # check that the scans list contains one scans
+    scans_tsv = make_bids_filename(
+        subject=subject_id, session=session_id, suffix='scans.tsv',
+        prefix=op.join(output_path, 'sub-01/ses-01'))
+    assert op.exists(scans_tsv)
+    if op.exists(scans_tsv):
```

Comment:
you just have an assert one line above. Do you need this then?

Reply:
oh yeah, I had this in for when I was doing some testing to make sure my paths were correct and that the scans file actually existed in the location I was constructing. I kept it however because my logic was that if somehow the scans file doesn't exist then the next check won't be run, but I realise now that the bids-validator will surely fail if it doesn't find the scans file anyway, so this is redundant. I'll remove the two instances of it

Reply:
@monkeyman192 the (unnecessary) check still seems to be there?